package.json

```diff
@@ -20,6 +20,7 @@
         "test": "jest --config ./scripts/jest/root.config.js",
         "test:debug": "node --inspect node_modules/.bin/jest --config ./scripts/jest/root.config.js --runInBand --watch",
         "test:ci": "yarn test --no-cache --coverage --runInBand",
+        "test:karma": "nx test @lwc/integration-karma",
```

Comment:
On its own, this is a good change... it's a nice shorthand, and we were being inconsistent before.